README.md

```diff
@@ -22,7 +22,21 @@ Install the following dependencies:
 
 ### Build and Run
 
-You can build and run this Docker image locally with the following steps:
+Make sure that Docker is running. Run the following command to launch the Apache Spark shell:
```

Comment:
> Please see the requested changes.
> 
> Additionally, I honestly don't see the need or use case for this at all. I get the frustration with the docker hub issue, but that has since been resolved (#26).

#26 has been resolved by removing the image from Dockerhub, correct?

The purpose of this PR is
* select the right `--jars` and `--py-files` automatically
* make the `docker run` command more memorable - basically, start without arguments to get a Spark shell, add `pyspark` for PySpark

The idea came because the JARs referenced in the docs were not in the container, so starting the container by the instructions provided didn't work. The script in this PR would still work because it would select the JARs available.

But I agree, not strictly necessary and a matter of taste.

Let me update the README, since Dockerhub images are no longer available.

Then feel free to just close the PR, up to you, I'm fine with whatever you decide. 👌🏽